hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -1274,7 +1274,7 @@ object TestHoodieSparkSqlWriter {
 
     // NOTE: Hudi doesn't support Orc in Spark < 3.0
     //       Please check HUDI-4496 for more details
-    val targetScenarios = if (HoodieSparkUtils.gteqSpark3_0) {
+    val targetScenarios = if (HoodieSparkUtils.gteqSpark3_3) {
```

Comment:
Let's add `TODO(HUDI-WXYZ)` for easy tracking of fixing these in a follow-up.

In this particular case, once Spark 2 is removed, there is no need to check Spark version.